openwisp_firmware_upgrader/admin.py

```diff
@@ -111,13 +111,7 @@ def organization(self, obj):
 
     organization.short_description = _('organization')
 
-    def reversion_register(self, model, **kwargs):
-        if model == FirmwareImage:
-            kwargs['follow'] = (
-                *kwargs['follow'],
-                'build',
-            )
-        return super().reversion_register(model, **kwargs)
+    # removed overriding reversion_register as recovery operation is no longer desired
```

Comment:
You can also remove this comment

Reply:
@AbhigyaShridhar can you proceed with this?

Reply:
Hi @pandafy , I made the requested changes. The branch was already up to date with master though. Please let me know if something else needs to be done. 

Apologies for the late reply, was facing some health issues in the past week.

Reply:
> Hi @pandafy , I made the requested changes. The branch was already up to date with master though. Please let me know if something else needs to be done.

Thank you for the update! 

> Apologies for the late reply, was facing some health issues in the past week.

No worries, get well soon! 😊